js/utils/dom.js

```diff
@@ -10,6 +10,12 @@ function isScrolledIntoView(el, scrollContainer) {
   return isVisible;
 }
 
+// https://developer.mozilla.org/en-US/docs/Web/API/Node/contains
```

Comment:
I saw in the codebase that jQuery offers this functionality... so we could probably remove this function at some point.